<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewPointSet</class>
 <widget class="QDialog" name="DialogNewPointSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Point Set</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter the name of the new point set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonControlPoint">
       <property name="text">
        <string>Control points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonWayPoint">
       <property name="text">
        <string>Way points</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelSelect">
     <property name="text">
      <string>Select template volume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxTemplate"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewPointSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewPointSet</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOK()</slot>
 </slots>
</ui>
